docs/pages/setup/guides/ec2-tags.mdx

```diff
@@ -4,131 +4,94 @@ description: How to set up Teleport Node labels based on EC2 tags
-- The following software installed on your Teleport Node: `curl`, `python`, and
-  the `aws` CLI, which comes from the `awscli` Python package.
+```bash
+$ tsh ls
+Node Name            Address        Labels                                                                                                                  
+-------------------- -------------- ----------------------------------------------------------------------------------------------------------------------- 
+fakehost.example.com 127.0.0.1:3022 env=example,hostname=ip-172-31-53-70,aws/Name=atburke-dev,aws/TagKey=TagValue,aws/TeleportHostname=fakehost.example.com
+```
 
-## Step 1/3. Deploy the script
+<Notice type="note">
+  For services that manage multiple resources (such as the database service), each resource will receive the
+  same labels from EC2.
+</Notice>
 
-You’ll need a script on your EC2 instance that can query the AWS API and get the
-values of your instance's tags for you. The Teleport Node will then use these
-values to execute RBAC rules.
+## Prerequisites
 
-Here’s one script you can use:
+<Tabs>
+<TabItem scope={["oss"]} label="Self-Hosted">
 
-```code
-#!/bin/bash
-if [[ "$1" == "" ]]; then
-	echo "Usage: $(basename $0) <tag>"
-	exit 1
-fi
-TAG_NAME=$1
-
-IMDS_TOKEN=$(curl -sS -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 300")
-IMDS_TOKEN_HEADER="-H \"X-aws-ec2-metadata-token: ${IMDS_TOKEN}\""
-INSTANCE_ID=$(curl -sS "${IMDS_TOKEN_HEADER}" http://169.254.169.254/latest/meta-data/instance-id)
-REGION=$(curl -sS "${IMDS_TOKEN_HEADER}" http://169.254.169.254/latest/meta-data/placement/availability-zone | sed -e "s:\([0-9][0-9]*\)[a-z]*\$:\\1:")
-TAG_VALUE="$(aws ec2 describe-tags --filters "Name=resource-id,Values=$INSTANCE_ID" "Name=key,Values=$TAG_NAME" --region $REGION --output=text | cut -f5)"
-
-if [[ "${TAG_VALUE}" == "" ]]; then
-	echo "<null>"
-else
-	echo $TAG_VALUE
-fi
-```
+- A Teleport cluster running on an AWS EC2 instance For details on setting this up, see [Getting
+  Started on a Linux Server](../../getting-started/linux-server.mdx).
 
-Save this script to `/usr/local/bin/get-tag.sh` on your EC2 instance.
-Run the command below to make it executable:
+  See [Installation](../../installation.mdx) for details.
 
-```code
-$ chmod +x /usr/local/bin/get-tag.sh
-```
+</TabItem>
+<TabItem
+  scope={["enterprise"]} label="Enterprise">
 
-## Step 2/3. Set up an IAM role
+- A Teleport cluster running on an AWS EC2 instance For details on setting this up, see our
+  [Enterprise getting started guide](../../enterprise/getting-started.mdx).
 
-Grant your EC2 instance an IAM role that will allow it to query tag values for EC2 instances.
+</TabItem>
+</Tabs>
 
-Here’s an example policy which you can add to an IAM role:
+<Admonition type="note" title="Note">
+  Instances that are not running on the [Nitro system](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-types.html#ec2-nitro-instances)
```

Comment:
This double negative is a little big confusing.

Instead of saying "instances **not** running on Nitro will **not**..." can you just describe the behavior of instances that are running on nitro?